npm/registry.go

```diff
@@ -112,5 +114,11 @@ func GetVersions(name string) []Version {
 		}
 	}
 
-	return versions
+	// get latest version according to npm
+	latest, ok := r.DistTags["latest"]
+	if !ok {
+		panic(fmt.Sprintf("no latest tag for npm package %s", name))
```

Comment:
I get a `panic: no latest tag for npm package angularjs-ie8-build` error when running the `autoupdate` tool. Are you experiencing the same?

I have changed a few things, but I don't think they should affect this part of the code: https://github.com/cdnjs/tools/compare/master...klausenbusk:sparse-checkout

Reply:
Yeah, we are seeing the same -- we've notified NPM and are waiting to see if they'll be quick in fixing it. If not, we'll look at add some logic that cleanly skips packages that fail to update.

Tracking in https://github.com/cdnjs/tools/issues/52